lib/clearance/configuration.rb

```diff
@@ -1,23 +1,94 @@
+
+    # A lambda called to set the remember token cookie expires attribute.
+    # The lambda accepts the collection of cookies as an argument which
+    # allows for changing the expiration according to those cookies.
+    # This could be used, for example, to set a session cookie unless
+    # a `remember_me` cookie was also present. By default, cookie expiration
+    # is one year. For more on cookie expiration see
+    # [RFC6265](http://tools.ietf.org/html/rfc6265#section-5.2.1).
+    # @return [Lambda]
+    attr_accessor :cookie_expiration
+
+    # The name of Clearance's remember token cookie.
+    # Defaults to `remember_token`.
+    # @return [String]
+    attr_accessor :cookie_name
+
+    # Controls which paths the remember token cookie is valid for.
+    # Defaults to `"/"` for the entire domain. For more, see
+    # [RFC6265](http://tools.ietf.org/html/rfc6265#section-5.1.4).
+    # @return [String]
+    attr_accessor :cookie_path
+
+    # Controls whether the  HttpOnly flag should be set on the remember token
+    # cookie. Defaults to `false`. If `true`, the cookie will not be made
+    # available to JavaScript. For more see
+    # [RFC6265](http://tools.ietf.org/html/rfc6265#section-5.2.6).
+    # @return [Boolean]
+    attr_accessor :httponly
+
+    # Controls the address the password reset email is sent from.
+    # Defaults to reply@example.com.
+    # @return [String]
+    attr_accessor :mailer_sender
+
+    # The password strategy to use when authenticating and setting passwords.
+    # Defaults to `Clearance::PasswordStrategies::BCrypt`.
+    # @return [Class #authenticated? #password=]
+    attr_accessor :password_strategy
+
+    # The default path Clearance will redirect signed in users to.
+    # Defaults to `"/"`. This can often be overridden for specific scenarios by
+    # overriding controller methods that rely on it.
+    # @return [String]
+    attr_accessor :redirect_url
+
+    # Set to `false` to disable Clearance's built-in routes.
+    # Defaults to `true`. When set to false, your app is responsible for all
+    # routes. You can dump a copy of Clearance's default routes with
+    # `rails generate clearance:routes`.
+    # @return [Boolean]
+    attr_writer :routes
+
+    # Controls the secure setting on the remember token cookie.
+    # Defaults to `false`. When set, the browser will only send the
+    # cookie to the server over HTTPS. You should set this value to true in
+    # live environments to prevent session hijacking. For more, see
+    # [RFC6265](http://tools.ietf.org/html/rfc6265#section-5.2.5).
+    # @return [Boolean]
+    attr_accessor :secure_cookie
+
+    # The array of sign in guards to run when signing a user in.
+    # Defaults to an empty array. Sign in guards respond to `call` and are
+    # initialized with a session and the current stack. Each guard can decide
+    # to fail the sign in, yield to the next guard, or allow the sign in.
+    # @return [Array #call]
```

Comment:
I'm trying to say it's an array of things that respond to `#call`. But it doesn't actually seem to work the way I'm lead to believe it works here: http://yardoc.org/types.html